generate.sh

```diff
@@ -52,7 +52,7 @@ if [ -r source/wordpress/wp-includes/Requests/Cookie/Jar.php ]; then
     )
     for METHOD in "${REQUESTS_V1_METHODS[@]}"; do
         # Get the line number where the method is defined.
-        LINE="$(php -r "require 'wordpress-stubs.php'; print (new ReflectionMethod('${METHOD}'))->getStartLine();")"
+        LINE="$(php -d error_reporting="E_ERROR" -r "require 'wordpress-stubs.php'; print (new ReflectionMethod('${METHOD}'))->getStartLine();")"
```

Comment:
releasing is peaceful on php 7.4

Reply:
Whoops, didn't mean to commit that. I suppose the script automatically stages all files and not just the stubs file.

FWIW I was using PHP 8.3 and got PHP warnings when loading the stubs file from older WordPress releases here.

Reply:
The WordPress ecosystem is not PHP 8 compatible.